doc/releases/release-notes-4.0.rst

```diff
@@ -56,6 +56,17 @@ Deprecated in this release
 
 * The :ref:`kscan_api` subsystem has been marked as deprecated.
 
+* Starting from this release the library is being marked as deprecated (:github:`79566`).
```

Comment:
`the library`? Maybe say which? :slightly_smiling_face:

Reply:
Ops, I copied that from the `migration-guide` but in that document this bullet point is in a TinyCrypt specific section so the library is implicit